device-types/Arista/DCS-7280SE-68-F.yaml

```diff
@@ -0,0 +1,130 @@
+---
+manufacturer: Arista
+model: DCS-7280SE-68-F
+part_number: DCS-7280SE-68-F
+slug: arista-dcs-7280se-68-f
+u_height: 1
+is_full_depth: true
+weight: 22.30
+weight_unit: lb
+airflow: front-to-rear
+comments: '[Arista 7280E Data Sheet](https://people.ucsc.edu/~warner/Bufs/7280SE-Datasheet.pdf)'
+front_image: true
+rear_image: true
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: Management1
+    type: 1000base-t
+    mgmt_only: true
+  - name: Ethernet1
+    type: 10gbase-x-sfpp
+  - name: Ethernet2
+    type: 10gbase-x-sfpp
+  - name: Ethernet3
+    type: 10gbase-x-sfpp
+  - name: Ethernet4
+    type: 10gbase-x-sfpp
+  - name: Ethernet5
+    type: 10gbase-x-sfpp
+  - name: Ethernet6
+    type: 10gbase-x-sfpp
+  - name: Ethernet7
+    type: 10gbase-x-sfpp
+  - name: Ethernet8
+    type: 10gbase-x-sfpp
+  - name: Ethernet9
+    type: 10gbase-x-sfpp
+  - name: Ethernet10
+    type: 10gbase-x-sfpp
+  - name: Ethernet11
+    type: 10gbase-x-sfpp
+  - name: Ethernet12
+    type: 10gbase-x-sfpp
+  - name: Ethernet13
+    type: 10gbase-x-sfpp
+  - name: Ethernet14
+    type: 10gbase-x-sfpp
+  - name: Ethernet15
+    type: 10gbase-x-sfpp
+  - name: Ethernet16
+    type: 10gbase-x-sfpp
+  - name: Ethernet17
+    type: 10gbase-x-sfpp
+  - name: Ethernet18
+    type: 10gbase-x-sfpp
+  - name: Ethernet19
+    type: 10gbase-x-sfpp
+  - name: Ethernet20
+    type: 10gbase-x-sfpp
+  - name: Ethernet21
+    type: 10gbase-x-sfpp
+  - name: Ethernet22
+    type: 10gbase-x-sfpp
+  - name: Ethernet23
+    type: 10gbase-x-sfpp
+  - name: Ethernet24
+    type: 10gbase-x-sfpp
+  - name: Ethernet25
+    type: 10gbase-x-sfpp
+  - name: Ethernet26
+    type: 10gbase-x-sfpp
+  - name: Ethernet27
+    type: 10gbase-x-sfpp
+  - name: Ethernet28
+    type: 10gbase-x-sfpp
+  - name: Ethernet29
+    type: 10gbase-x-sfpp
+  - name: Ethernet30
+    type: 10gbase-x-sfpp
+  - name: Ethernet31
+    type: 10gbase-x-sfpp
+  - name: Ethernet32
+    type: 10gbase-x-sfpp
+  - name: Ethernet33
+    type: 10gbase-x-sfpp
+  - name: Ethernet34
+    type: 10gbase-x-sfpp
+  - name: Ethernet35
+    type: 10gbase-x-sfpp
+  - name: Ethernet36
+    type: 10gbase-x-sfpp
+  - name: Ethernet37
+    type: 10gbase-x-sfpp
+  - name: Ethernet38
+    type: 10gbase-x-sfpp
+  - name: Ethernet39
+    type: 10gbase-x-sfpp
+  - name: Ethernet40
+    type: 10gbase-x-sfpp
+  - name: Ethernet41
+    type: 10gbase-x-sfpp
+  - name: Ethernet42
+    type: 10gbase-x-sfpp
+  - name: Ethernet43
+    type: 10gbase-x-sfpp
+  - name: Ethernet44
+    type: 10gbase-x-sfpp
+  - name: Ethernet45
+    type: 10gbase-x-sfpp
+  - name: Ethernet46
+    type: 10gbase-x-sfpp
+  - name: Ethernet47
+    type: 10gbase-x-sfpp
+  - name: Ethernet48
+    type: 10gbase-x-sfpp
+# QSFP28(100G) ports can be broke into 4x25 with a model of [49-54]/[1-4]
+  - name: Ethernet49/1
+    type: 100gbase-x-qsfp28
+  - name: Ethernet50/1
+    type: 100gbase-x-qsfp28
+power-ports:
+  - name: PS1
```

Comment:
Looking at the images provided these are hot swappable PSUs, therefore they need to modeled under `module-bays` with the required PSU models added if they don't already exist

Reply:
Updated